src/actions/attack.cpp

```diff
@@ -157,6 +155,10 @@ battle_context_unit_stats::battle_context_unit_stats(const unit& u,
 
 	// Handle plague.
 	unit_ability_list plague_specials = weapon->get_specials("plague");
+	unit_ability_list alt_plague_specials = weapon->list_ability("plague");
```

Comment:
why not also using get_special_ability here?

Reply:
@gfgtdf here the value returned is not nulerical but a sring, it is necessary to etablish a priority for avoid conflict with two diffrent value, if and like i get_special.empty for using abilitie instead, i prefer put list_ability instead of get_special_ability, but yes i could using that.

Reply:
hmm opklay i guess its not that important, i still think just using get_special_ability would be better